### PR TITLE
PWN-9339 - main and crypto screens analytics added

### PR DESCRIPTION
**Pull request by kabramovp2p**
## Jira Ticket

https://p2pvalidator.atlassian.net/browse/PWN-9339

## Description of Work

PWN-9339 - main and crypto screens analytics added

### CODE REVIEW

File: app/src/main/java/org/p2p/wallet/home/ui/container/MainContainerFragment.kt [hunk 1]
```diff
@@ -226,7 +213,8 @@ class MainContainerFragment :
             }
             else -> {
                 when (clickedTab) {
-                    ScreenTab.MY_CRYPTO_SCREEN -> presenter.logHomeOpened()
+                    ScreenTab.WALLET_SCREEN -> presenter.logWalletOpened()
+                    ScreenTab.MY_CRYPTO_SCREEN -> presenter.logWalletOpened()
```
Comment (eduardmaximovich) [inline app/src/main/java/org/p2p/wallet/home/ui/container/MainContainerFragment.kt:217]:
```suggestion
                    ScreenTab.MY_CRYPTO_SCREEN -> presenter.logCryptoOpened()
```

Comment (kabramovp2p) [reply app/src/main/java/org/p2p/wallet/home/ui/container/MainContainerFragment.kt:217]: krasava sps

File: app/src/main/java/org/p2p/wallet/home/ui/container/MainContainerFragment.kt [hunk 2]
```diff
@@ -226,7 +213,8 @@ class MainContainerFragment :
             }
             else -> {
                 when (clickedTab) {
-                    ScreenTab.MY_CRYPTO_SCREEN -> presenter.logHomeOpened()
+                    ScreenTab.WALLET_SCREEN -> presenter.logWalletOpened()
```
Comment (gslevinkov) [inline app/src/main/java/org/p2p/wallet/home/ui/container/MainContainerFragment.kt:216]: are we sure that we want to add some analytic methods to presenter? 
maybe inject analytics just here and remove this proxy methods from presenter?

File: app/src/main/java/org/p2p/wallet/home/ui/wallet/WalletModule.kt [hunk 3]
```diff
@@ -18,6 +18,21 @@ object WalletModule : InjectionModule {
         factoryOf(::StrigaOnRampConfirmedHandler)
         factoryOf(::StrigaOnRampClickHandler)
         factoryOf(::StrigaBannerClickHandler)
-        factoryOf(::WalletPresenter) bind WalletContract.Presenter::class
+        factory {
+            WalletPresenter(
```
Comment (gslevinkov) [inline app/src/main/java/org/p2p/wallet/home/ui/wallet/WalletModule.kt:22]: wow, presenter is big again

**Review by gslevinkov — APPROVED** [review]

**Review by 1sakov — APPROVED** [review]
When you'll add unit tests for these screens? @kabramovp2p

File: app/src/main/java/org/p2p/wallet/home/ui/wallet/WalletModule.kt (hunk 3 above)
Comment (1sakov) [reply app/src/main/java/org/p2p/wallet/home/ui/wallet/WalletModule.kt:22]: exactly, we need to move striga stuff

**Review by eduardmaximovich — APPROVED** [review]

**Pull request merged**